### ISSUE TITLE
Prisma in non-awaited async function kills event loop

### ISSUE THREAD

**Issue by petter-kaspersen** [oven-sh/bun#11334]
### What version of Bun is running?

1.1.8+89d25807f

### What platform is your computer?

Linux 5.15.146.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

Write a simple code snippet that calls Prisma. Eg.

```js
async function doSomething() {
     console.log("Entry!");
     
     await prisma.someModel.findMany({});
     
     console.log("Exit!")
}
```

Then call this function by a non-awaited call.

```js
doSomething()
``` 

What happens is that the call to Prisma just fizzles out, and it exits the promise at that location. This is not how both regular Node and TS-Node behaves, as they will successfully completely call everything.

### What is the expected behavior?

The entry log should be called, the Prisma function executed, and the exit log should be called

### What do you see instead?

Only the entry log is called, no Prisma call to the database takes place

### Additional information

Does not happen with any other async call (from limited testing), only Prisma

**Comment by Jarred-Sumner** [comment]
I think @190n fixed this recently

**Comment by 190n** [comment]
@petter-kaspersen Can you try again after `bun upgrade --canary`? This should have been fixed in #14068.

**Comment by 190n** [comment]
I'm going to close this, since we fixed the direct cause when it was reported by another user, but if you still run into this issue on the latest version of Bun then you can re-open it.
